neural_compressor/torch/quantization/autotune.py

```diff
@@ -0,0 +1,67 @@
+# Copyright (c) 2023 Intel Corporation
```

Comment:
The `autotune.py` has two duplicates.
https://github.com/intel/neural-compressor/blob/xinhe/rename/neural_compressor/torch/autotune.py
https://github.com/intel/neural-compressor/blob/xinhe/rename/neural_compressor/torch/quantization/autotune.py